assets/model_monitoring/components/src/generation_safety_quality/annotation_compute_metrics/run.py

```diff
@@ -109,14 +129,30 @@ def run():
                             _calculate_passrate(histogram_df, metric_name),
                             full_pass_rate_metric_name,
                             passrate_threshold,
+                            ""
                         )
                     ],
-                    histogram_df.schema,
+                    metadata_schema,
                 )
             aggregated_metrics_df = aggregated_metrics_df.union(metric_df)
         if full_per_instance_score_metric_name not in input_metric_names:
-            aggregated_metrics_df = aggregated_metrics_df.filter(col("metric_name")
+            aggregated_metrics_df = aggregated_metrics_df.filter(col(METRIC_NAME_COLUMN)
                                                                  != full_per_instance_score_metric_name)
+        else:
+            threshold = _get_per_instance_threshold(histogram_df, metric_name)
+            threshold_row = spark.createDataFrame(
+                    [
+                        (
+                            "",
+                            "",
```

Comment:
Sorry, I am not familiar with the per instance score metrics, but I see both "group name" and "metric value" are empty here. Just curious how this works without metric values.

Reply:
do you mean how does that work in the evaluate component ? Mathieu wrote that part and just gave me the output he wanted.

Reply:
Synced offline and it seems these are some metrics not for evaluation metric component's consumption, but for UI instead. Hence they are skipped in evaluation metrics.